### PR TITLE
issue: 770746 Add csbuild support into jenkins

### PR DESCRIPTION
**Pull request by igor-ivanov**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/434/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/450/ for details (Mellanox internal link).

File: src/vma/vma_extra.h [hunk 1]
```diff
@@ -36,6 +36,9 @@
 
 #include <stddef.h>
 #include <stdint.h>
+
+#include "utils/bullseye.h"
```
Comment (rosenbaumalex) [inline src/vma/vma_extra.h]: @igor-ivanov, this `#include` is problematic, need to find a different way.
We install `vma_extra.h` with libvma.rpm and allow application to use this header file and later running/loading with libvma.so itself.
But these application will not see the rest of the VMA tree and header files like the `utils/bullseye.h` in order to compile correctly. Need to keep here ONLY system include files.

Comment (igor-ivanov) [inline src/vma/vma_extra.h]: i see, will find something

Comment (rosenbaumalex) [inline src/vma/vma_extra.h]: @igor-ivanov I believe we can remove the BullseyeCoverage #pragma from this file now.
@liranoz12 added code that calls the `vma_get_api()` in commit https://github.com/Mellanox/libvma/commit/5f8ac759b0bc817691c03d45af7fddda928e58af, so in fact it is in use and can be easily covered with new test calling `vma_stats -S <fd>`

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/441/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/457/ for details (Mellanox internal link).

**Pull request merged**